tests/scripts/check-names.sh

```diff
@@ -70,7 +70,7 @@ NL='
 '
 sed -n 's/MBED..._[A-Z0-9_]*/\'"$NL"'&\'"$NL"/gp \
     $HEADERS library/*.c \
-    | grep MBEDTLS | sort -u > _MBEDTLS_XXX
+    | grep MBEDTLS | grep -v MBED_OS_MBEDTLS | sort -u > _MBEDTLS_XXX
```

Comment:
It is necessary to simply exclude `MBED_OS_MBEDTLS`, otherwise the script will think it is a typo and the travis CI always fails.